unity-renderer/Assets/DCLPlugins/EmoteAnimations/EmoteAnimationsTracker.cs

```diff
@@ -35,20 +36,17 @@ public EmoteAnimationsTracker(DataStore_Emotes dataStore, EmoteAnimationLoaderFa
             this.emotesCatalogService = emotesCatalogService;
             this.dataStore.animations.Clear();
 
-            InitializeEmbeddedEmotes();
-            InitializeEmotes(this.dataStore.emotesOnUse.GetAllRefCounts());
-
-            this.dataStore.emotesOnUse.OnRefCountUpdated += OnRefCountUpdated;
+            AsyncInitialization();
         }
 
-        private void InitializeEmbeddedEmotes()
+        private async void AsyncInitialization()
```

Comment:
to `UniTask` with `cancelationToken` (that will be passed to `await` in  line 49)

Reply:
Done!